content/en/tutorials/touch-joypad.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Touchscreen Joypad Controls
+layout: tutorial-page.hbs
+tags: input,ui
+thumb: https://s3-eu-west-1.amazonaws.com/images.playcanvas.com/projects/12/1007506/2F5E56-image-75.jpg
+---
+
+<iframe loading="lazy" src="https://playcanv.as/p/kvE0iJWc/" title="Touchscreen Joypad Controls"></iframe>
+
+[Click here to see the project][project-link].
+
+## Overview
+
+This tutorial shares a Touchscreen Joypad library that can be installed and customized for any project that needs touch screen controls.
+
+We recommend reading about the [PlayCanvas UI system][playcanvas-ui] if you are not familiar with it as it to render and position the controls.
+
+![][preview]
+
+## How to install
+
+Open the [example project][project-link], right click on the folder 'touch-joypad' and click on 'Copy'.
+![][copy-folder]
+
+Open your project, right click in the assets panel and click on 'Paste'
+![][paste-folder]
+
+## Adding your joystick
+
+In the folder we've just pasted, open the 'templates' folder and there will be 4 preconfigured templates for the joysticks to cover the most common use cases.
+
+* **Left/Right Fixed Touch Joystick** - A fixed joystick that is anchored to the bottom left/right of the screen corners. Recommended for quick reaction games where an instant response is expected such as a retro platforming game.
+* **Left/Right Half Touch Joystick** - A joystick where the input area the left/right half of the screen and the joystick will move to where you touch in that area and them requires dragging for movement. Recommended for when 'pushing' or 'pulling' an object in the application.
+
+For this example, let's use the 'Left Half Touch Joystick' template.
+
+Create a 2D screen Entity and [add the template][add-template-docs] as a child of the screen Entity.
+
+![][adding-left-half-joystick]
+
+The joystick is made of three Entities:
+
+- Input area (outlined in red)
+- Base (outlined in blue)
+- Nub (outlined in blue)
+
+![][joystick-layout]
+
+The input area will listen for touch and mouse (for debugging purposes) events and also stops the propagation of the event.
+
+This means that any UI Elements under this and any [pc.Mouse][pc-app-mouse] or [pc.Touch][pc-app-touch] events will not be fired if this area is interacted with first.
```

Comment:
should it look like this?

<img width="784" alt="Screenshot 2022-12-05 at 15 47 54" src="https://user-images.githubusercontent.com/59932779/205680606-8e8b8a20-826c-452f-85d0-072f82b40763.png">